app/views/help/email_filtering.md

```diff
@@ -4,7 +4,7 @@ Yes. You’ll need to set up a filter in Gmail.
 
 ## First, select the C2 request messages to filter.
 
-1. Enter the following into the search bar at the top of your inbox: `from:communicart.sender@gsa.gov`
+1. Enter the following into the search bar at the top of your inbox: `from:c2.sender@gsa.gov`
```

Comment:
ah yes I can return it to the old one. Did a global find an replace and thought I'd reverted all instances of `communicart.sender` but I see that I missed this one.